bin/journey-test.sh

```diff
@@ -181,7 +181,7 @@ start_x_api() {
   pushd $xapi_home
 
   gem install bundler
-  bundle install
+  bundle install -j4
```

Comment:
How did you decide on a value of 4 here?

Also, let's use `--jobs` if we can; makes it easier to parse later on :)